scripts/build-native.js

```diff
@@ -43,9 +43,13 @@ async function build() {
 // This forces Clang/LLVM to be used as a C compiler instead of GCC.
 // This is necessary for cross-compilation for Apple Silicon in GitHub Actions.
 function setupMacBuild() {
-  let xcodeDir = execSync('xcode-select -p | head -1', {encoding: 'utf8'}).trim();
-  let sysRoot = execSync('xcrun --sdk macosx --show-sdk-path', {encoding: 'utf8'}).trim();
-  process.env.CC = `${xcodeDir}/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang`;
-  process.env.CXX = `${xcodeDir}/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++`;
+  let xcodeClangPath = execSync('xcrun -f clang', {
+    encoding: 'utf8',
+  }).trim();
+  let sysRoot = execSync('xcrun --sdk macosx --show-sdk-path', {
+    encoding: 'utf8',
+  }).trim();
+  process.env.CC = xcodeClangPath;
+  process.env.CXX = `${xcodeClangPath}++`;
```

Comment:
@mischnic do you find this line a bit flaky? it works

Reply:
I think we should also call `xcrun -f clang++` for that. This kind of assumption is exactly why it was broken before 😄

Reply:
@mischnic lol the cost of "being DRY" :) done

Reply:
@mischnic the first 2 workflows failed, I think re-triggering them will resolve it (there's no sense in failing for ts type validation issues as the change is on a js file :) + I ran the tests locally just to be 100% sure there's nothing I'm missing.

Reply:
👍  Thanks

You can ignore the failures, CI has been flakey for some time now.